container/secrets.go

```diff
@@ -81,5 +86,8 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	if _, err := os.Stat(overrideDir); err == nil {
```

Comment:
Hmm...I was thinking more like *individual files* with the same name override (systemd like), whereas this seems to be "if /etc exists, it wins entirely", so users would have to copy over data from `/usr` that they want to keep.